samples/sensor/sx9500/src/main.c

```diff
@@ -76,7 +76,10 @@ void main(void)
 	struct device *dev;
 
 	dev = device_get_binding("SX9500");
-	printk("device is %p, name is %s\n", dev, dev->config->name);
```

Comment:
Same as above; would it not be better to leave this line in just move it to after the error trapping?

Reply:
Yes, I will change it.

Reply:
@WilliamGFish Do you know how to add labels? I have check zephyr-dev mail list but still no clue. I suppose need authority. Anyway, I think some of my patches are samples related, it's better to be merged before 1.14 official released to give the beginner a good first impression. Thanks.

Reply:
You're correct it is permissions thing, don't worry about being a newbie because so am I.